roku_dev_cli/roku_dev_cli.py

```diff
@@ -588,6 +790,7 @@ def main():
     parser.add_argument('-v', '--version', action='version', version='roku-dev-cli {}'.format(__version__))
     parser.add_argument('-t', action='store_true', help="display timestamp in front of each line")
     parser.add_argument('-n', action='store_true', help="track node usage over time")
+    parser.add_argument('--report_rendezvous', action='store_true', help='reports telnet rendezvous events to file')
```

Comment:
Suggest making this `'--report-rendezvous'` (no underscore) for consistency with other command line tools and options. No other code change necessary since `argparse` will automatically represent the option as `args.report_rendezvous`.

Reply:
Good callout.  Will change.